robottelo/ui/computeresource.py

```diff
@@ -191,6 +195,17 @@ def set_values(self, **kwargs):
         if template is not None:
             self.set_value(template_key, template)
             del kwargs[template_key]
+        memory = kwargs.get('memory')
+        if memory is not None:
+            # to make the memory value take effect we have to set it's hidden
+            # input value
+            memory_hidden_input = self.page.wait_until_element_exists(
+                self.memory_locator_hidden)
+            # memory must be in bytes, 1 MB = 1024*1024 = 1048576 bytes
+            self.page.browser.execute_script(
```

Comment:
sorry, but can you provide screenshot?
Are you trying to perform action that real user cannot do?

Reply:
Seems we also cannot do what the user can do as, seems there is a script that handle that field and copy the value to the hidden field , eg: when send keys by selenium to memory field  the hidden field is not updated and the value is not updated.
(of course we can see the  new value in memory field but that value is not saved as not detected )
the only way that I found is to explicitly set that hidden filed value.

Reply:
hmm, I think it should be investigated more as `send_keys` does exactly the same as when user input data from keyboard. Please, look what events are called and what trigger copy paste from real field to hidden.
Sorry, but proposed solution is not acceptable from WebUI test automation perspective

Reply:
@oshtaier think I found how to explicitly fire that event, without setting the hidden value, will make the necessary changes